GUI_environment.yaml

```diff
@@ -0,0 +1,20 @@
+name: annotate_env
```

Comment:
The other environment yaml has the name of the environment as `sam2-env`. For consistency sake, I suggest we rename this to `annotate-env`. If you agree, all `README.md` items should also be updated.